pkgs/development/libraries/wlroots/default.nix

```diff
@@ -20,14 +21,16 @@
 , seatd
 , vulkan-loader
 , glslang
+, libliftoff
+, libdisplay-info
 , nixosTests
 
 , enableXWayland ? true
 , xwayland ? null
 }:
 
 let
-  generic = { version, hash, extraBuildInputs ? [ ], extraNativeBuildInputs ? [ ], extraPatch ? "" }:
+  generic = { version, hash, extraBuildInputs ? [ ], extraNativeBuildInputs ? [ ], extraPatch ? [ ], extraPostPatch ? "" }:
```

Comment:
I wouldn't name them all with extra as a prefix

Reply:
What is a better approach？

(They are used from https://github.com/NixOS/nixpkgs/pull/201453)